.dockerignore

```diff
@@ -1,5 +1,6 @@
-Dockerfile
-go.mod
-go.sum
-main.go
-Makefile
+*
+!.git/
```

Comment:
Why include the .git directory? That's an awful lot of extra data in the image for AFAICT little to no value.

Reply:
This is only copied to the builder image so https://github.com/mcrute/ses-smtpd-proxy/blob/master/Makefile#L6 works. The final container only has the built executable.

Reply:
Oh yeah, that's a good point. I still don't see any reason for it but since it's only in the build image I also have no objections to it being there.

Reply:
Without it, the version won't be set since that uses the git tags. Alternatively, that can be run by make and passed to Dockerfile as a parameter.